distuv/bernouilli.go

```diff
@@ -0,0 +1,111 @@
+package distuv
+
+import (
+	"math"
+	"math/rand"
+)
+
+// Bernoulli represents a random variable whose value is 1 with probability p and
+// value of zero with probability 1-P. The value of P must be between 0 and 1.
+type Bernoulli struct {
```

Comment:
for a few types in `gonum/*` we had a link to a wikipedia description of the mathematical concept.
should we follow the trend and add https://en.wikipedia.org/wiki/Bernoulli_distribution here as well?